Source/PropertyTools.Wpf/PropertyGrid/IPropertyGridControlFactory.cs

```diff
@@ -36,5 +36,7 @@ public interface IPropertyGridControlFactory
         /// <param name="tab">The tab.</param>
         /// <returns></returns>
         ContentControl CreateErrorControl(PropertyItem pi, object instance, Tab tab, PropertyControlFactoryOptions options);
+
+        FrameworkElement SetValidationErrorStyle(FrameworkElement control, PropertyControlFactoryOptions options);
```

Comment:
do we need the return value? we have not used "fluent" syntax in this library

Reply:
did not understand this, this is an interface and we can not return anything from here

Reply:
the method interface defines a `FrameworkElement` to be returned, could it be `void`?

Reply:
fixed.